File: manager/src/main/java/club/dreamccc/manager/controller/ZookeeperController.java
package club.dreamccc.manager.controller;

import org.apache.curator.framework.CuratorFramework;
import org.springframework.web.bind.annotation.RequestMapping;

import javax.annotation.Resource;


@RequestMapping("/zk")
public class ZookeeperController {

    @Resource
    CuratorFramework curatorFramework;

    @RequestMapping("/show")
    public Object show() throws Exception {

        byte[] bytes = curatorFramework.getData().forPath("/lock");

        return new String(bytes);
    }

}

File: manager/src/main/java/club/dreamccc/manager/controller/ZookeeperController.kt
package club.dreamccc.manager.controller

import org.apache.curator.framework.CuratorFramework
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import javax.annotation.Resource

@RestController
@RequestMapping("/zk")
class ZookeeperController {

    @Resource
    internal var curatorFramework: CuratorFramework? = null


    @RequestMapping("/show")
    fun show(): Any {

        val bytes = curatorFramework!!.data.forPath("/lock")

        return String(bytes)
    }

}
